apps/scraper-runner/src/main.ts

```diff
@@ -0,0 +1,79 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+import * as healthSiteScraper from '@foodmedicine/health-site-scraper';
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+
+/**
+ * Get the top percentage of an array
+ * @param arr an array sorted in descending order
+ * @param percent percentage of items to be returned
+ */
+function getTopPercentage(arr: any[], percent = 5): any[] {
```

Comment:
Look into TS generic function syntax to replace `any` with `T`.  Avoid using `any` where possible, as it provides the least value of all types

An example copied from online:
```
function getArray(items : any[] ) : any[] {
    return new Array().concat(items);
}
```
becomes
```
function getArray<T>(items : T[] ) : T[] {
    return new Array<T>().concat(items);
}
```

more info at https://www.typescriptlang.org/docs/handbook/generics.html#working-with-generic-type-variables